src/track/bpm.h

```diff
@@ -12,6 +12,8 @@ class Bpm final {
 public:
     static constexpr double kValueUndefined = 0.0;
     static constexpr double kValueMin = 0.0; // lower bound (exclusive)
+    static constexpr double kValueMinForTwoDecimals = 5000.0;
+    static constexpr double kValueMinForOneDecimal = 500.0;
```

Comment:
We should put here a good reference why 50 to 500 is a good BPM range. What is the border of the other tools? 
My Metronom has a range of 40 ...  208 Bpm
This is also confirmed in this German Wikipedia article:
https://de.wikipedia.org/wiki/Tempo_(Musik)#Metronomangaben

How about divide the Bpm until it is in the range of 30 .. 300 BPM?